recipes/hugo/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+export CGO_ENABLED=1
```

Comment:
If you set this flag, you want go-cgo instead of nocgo. Change the compiler and get rid of setting this flag manually